test/NuGet.Core.FuncTests/Dotnet.Integration.Test/PackCommandTests.cs

```diff
@@ -3775,6 +3790,11 @@ public void PackCommand_PackEmbedInteropPackage()
             }
         }
 
+        [Fact(Skip = "waiting for getting knownFrameworkReference, https://github.com/NuGet/Home/issues/8155")]
```

Comment:
Why add a duplicate definition? 

You can add the skip on the theory itself.

Reply:
Changed to theory skip and commented out "PlatformTheory" only.